specification/azsadmin/resource-manager/backup/Microsoft.Backup.Admin/preview/2018-09-01/BackupLocations.json

```diff
@@ -0,0 +1,325 @@
+    ],
+    "consumes": [
+        "application/json"
+    ],
+    "paths": {
+        "/subscriptions/{subscriptionId}/resourcegroups/{resourceGroupName}/providers/Microsoft.Backup.Admin/backupLocations": {
+            "get": {
+                "x-ms-examples": {
+                    "Returns the list of backup locations.": {
+                        "$ref": "./examples/BackupLocations/List.json"
+                    }
+                },
+                "description": "Returns the list of backup locations.",
+                "tags": [
+                    "BackupLocations"
+                ],
+                "operationId": "BackupLocations_List",
+                "parameters": [
+                    {
+                        "$ref": "Backup.json#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "Backup.json#/parameters/ResourceGroupParameter"
+                    },
+                    {
+                        "$ref": "Backup.json#/parameters/ApiVersionParameter"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "OK",
+                        "schema": {
+                            "$ref": "#/definitions/BackupLocationList"
+                        }
+                    }
+                },
+                "x-ms-pageable": {
+                    "nextLinkName": "nextLink"
+                }
+            }
+        },
+        "/subscriptions/{subscriptionId}/resourcegroups/{resourceGroupName}/providers/Microsoft.Backup.Admin/backupLocations/{location}": {
+            "get": {
+                "x-ms-examples": {
+                    "Returns a specific backup location based on name.": {
+                        "$ref": "./examples/BackupLocations/Get.json"
+                    }
+                },
+                "description": "Returns a specific backup location based on name.",
+                "tags": [
+                    "BackupLocations"
+                ],
+                "operationId": "BackupLocations_Get",
+                "parameters": [
+                    {
+                        "$ref": "Backup.json#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "Backup.json#/parameters/ResourceGroupParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/BackupLocationParameter"
+                    },
+                    {
+                        "$ref": "Backup.json#/parameters/ApiVersionParameter"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "OK",
+                        "schema": {
+                            "$ref": "#/definitions/BackupLocation"
+                        }
+                    },
+                    "404": {
```

Comment:
Please remove 404, when you specify 404 here, the SDK will eat out the exception. We have been standardizing on not to specify any error codes in the swagger